cloud/observability/promql-to-scrape/internal/metric.go

```diff
@@ -0,0 +1,18 @@
+package internal
+
+import "strings"
+
+const (
+	metricTypeHistogram = "histogram"
+	metricTypeCounter   = "count"
```

Comment:
This should be "counter" instead of "count"


https://github.com/prometheus/OpenMetrics/blob/main/specification/OpenMetrics.md#type